connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -455,6 +456,11 @@ private WorkerTask buildWorkerTask(ConnectorConfig connConfig,
                     internalKeyConverter, internalValueConverter);
             OffsetStorageWriter offsetWriter = new OffsetStorageWriter(offsetBackingStore, id.connector(),
                     internalKeyConverter, internalValueConverter);
+            for (Entry<String, Object> connectorProp : connConfig.originals().entrySet()) {
```

Comment:
`Map.Entry` is already utilized within this class, so it should be used again here rather than imported separately.